src/markdown/features/data-management.md

```diff
@@ -56,6 +55,16 @@ We have in the *users* collection
 }
 ```
 
+#### Update Many
+
+Update many document, give filter and update in body
+```js
+- PUT `${api.url}/app/colls/${coll}/docs/update}`
```

Comment:
We should use the Mongo function name as endpoint. We also should remove the  `docs/` from the function endpoints (we should do that for the find two but we need to keep compatibility...):

```suggestion
- PUT `${api.url}/app/colls/${coll}/updateMany`
```